frontend/src/concepts/pipelines/apiHooks/mlmd/useGetLinkedArtifactsByEvents.ts

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { usePipelinesAPI } from '~/concepts/pipelines/context';
+import { Artifact, Event, GetArtifactsByIDRequest } from '~/third_party/mlmd';
+import useFetchState, { FetchState, FetchStateCallbackPromise } from '~/utilities/useFetchState';
+
+// An artifact which has associated event.
+// You can retrieve artifact name from event.path.steps[0].key
+export interface LinkedArtifact {
+  event: Event;
+  artifact: Artifact;
+}
```

Comment:
DO you think we should have a MLMD types file somwhere?

Reply:
Yeah... I can create a file named `mlmdTypes.ts` and put the types into it.